superduperdb/db/mongodb/data_backend.py

```diff
@@ -35,7 +35,16 @@ def build_metadata(self):
         return MongoMetaDataStore(self.conn, self.name)
 
     def build_artifact_store(self):
-        return MongoArtifactStore(self.conn, f'_filesystem:{self.name}')
+        try:
+            return MongoArtifactStore(self.conn, f'_filesystem:{self.name}')
+        except TypeError as e:
+            from mongomock import MongoClient as MockClient
+
+            if isinstance(self.conn, MockClient):
```

Comment:
you can put this code in the starting, this way you won't need the try and expect thingy